client-data/board.css

```
@@ -113,7 +113,7 @@ html, body, svg {
 	max-width: 100%;
 }
 
-@media (hover: none) {
+@media (any-hover: none) or (any-pointer: coarse) {
```

```suggestion
@media (hover: none), (pointer: coarse) {
```

I don't think there is a `or` in CSS media queries. And using the data reported for the main input device is probably more appropriate.